src/pages/try.js

```diff
@@ -217,6 +240,9 @@ export default class Try extends Component {
         <div css={{ backgroundColor: accent, color: 'white' }}>
           <Header inverted />
         </div>
+        <center>
+          <span> <button onClick={this.handleClick} css={styles.ButtonStyle}>Create Tryit Link</button> </span>
```

Comment:
I'm thinking we should just debounce and then serialize the code into url every 100ms. Might be potentially distracting; let's see if that works though